### PR TITLE
Updates to EIPs 2464 and 2364.

### PR DESCRIPTION
**Pull request by timbeiko**
Combines the updates from https://github.com/ethereum/EIPs/pull/4367 and https://github.com/ethereum/EIPs/pull/4366, and addresses most comments from @MicahZoltu in a way which hopefully makes him happy, but recognizes the fact that these EIPs are already "ancient". Addressed all comments which were related to formatting/sections, but links are still present, and the specification has been left as is. 

Hopefully we can get a soft +1 on these before requiring Péter to review & approve.

### CODE REVIEW

**Comment by timbeiko** [comment]
Should have addressed your comments, @MicahZoltu! Would like @karalabe to approve before we merge given these are his EIPs and the modifications are a bit more extensive than I'd planned.

**Review by karalabe — APPROVED** [review]

**Comment by axic** [comment]
There's also the `Backwards Compatibility` section as a requirement according to EIP-1.

**Comment by MicahZoltu** [comment]
Both appear to have a backward compatibility section.

**Review by MicahZoltu — APPROVED** [review]

**Comment by MicahZoltu** [comment]
Apparently I don't have authority to approve this anymore!  See #4418 to fix that, in the meantime @lightclient or @axic approving should result in auto-merge.

**Comment by axic** [comment]
> Both appear to have a backward compatibility section.

Hmm, correct. The diff view was expanding weirdly yesterday.

File: EIPS/eip-2364.md [hunk 1]
```diff
@@ -40,15 +41,17 @@ Whenever two peers connect using the `eth/64` protocol, the updated `Status` mes
 
 ## Rationale
 
-##### EIP-2124 mentions advertising the `forkid` in the discovery protocol too. How does that compare to advertising in the `eth` protocol? Why is the redundancy needed?
+The specification is tiny since most parts are already specified in EIP-2124. `eth/63` is not specified as an EIP, but is maintained [here](https://github.com/ethereum/devp2p/blob/master/caps/eth.md).
```
Comment (axic) [inline EIPS/eip-2364.md:44]: Not sure why this needs to be here? Is this the way to make @MicahZoltu allow an external URL? :)

Comment (axic) [reply EIPS/eip-2364.md:44]: And can this URL be a tagged one, and not `master`?

Comment (MicahZoltu) [reply EIPS/eip-2364.md:44]: Ah, you are right, that should be removed.  It is not necessary for the specification.

**Pull request merged**